recipes/tf_object_detection/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "tf_object_detection" %}
+{% set version = "0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+source:
+  url: https://github.com/Maxar-Corp/ratpac/releases/download/od-{{ version }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 476eaf754c61c5afce39e36683bb4a7a6b7f9084e2bc89f075d58e294da27768
+
+build:
+  number: 0
+  skip: True  # [win]  # Skip all windows for now due to numpy/scipy version pinning issues
+  skip: True  # [win32]
```

Comment:
Also, we don't build win32 and so this is redundant with the above

Reply:
@scopatz on the recipe for [tensorflow-feedstock](https://github.com/conda-forge/tensorflow-feedstock/blob/master/recipe/meta.yaml), testing on `linux` and `OSX` has been is indicated to be challenging. The comments section says:

       Skip the import test on Linux as wheel file require a more recent
       version of GLIBC++ than the VM used to build and test package.
       Also skip on OSX due to OSX version requirements, see
       https://github.com/conda-forge/tensorflow-feedstock/pull/62

Reply:
@conda-forge/tensorflow I'm building recipe for the object_detection API by following the steps on `tensorflow-feedstock` recipe. please help us understand the issue with skipping test for `linux` and `osx`.